### PR TITLE
ApplePay Integration

### PR DESCRIPTION
**Pull request by mpetrenco**
### Added support for ApplePay payments.

**Note:**
For demo purposes a mocked JPResponse has been created. It bypasses the JudoKit response validation logic and sends the user to the payment summary screen, where the user can see the billing / shipping contact information. This code will be removed as soon as this functionality is showcased.

### CODE REVIEW

File: Source/ApplePay/ApplePayConfiguration.m [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+#pragma mark - Helper methods
+
+- (void)setupDefaults {
+    self.transactionType = TransactionTypePayment;
+    self.merchantCapabilities = MerchantCapability3DS;
+    self.shippingType = ShippingTypeShipping;
+    self.returnedContactInfo = ReturnedInfoBillingContacts;
+    self.supportedCardNetworks = @[@(CardNetworkVisa), @(CardNetworkMaestro), @(CardNetworkAMEX)];
```
Comment (stpopa) [inline Source/ApplePay/ApplePayConfiguration.m]: please remove from defaults "CardNetworkMaestro"

File: fastlane/Fastfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ platform :ios do
   
   lane :generate_code_quality_reports do
     # sh("cd .. && ./clang-format.sh lint") # check code format
-    scan(use_clang_report_name: true)     # run tests and generate reports
+    # scan(use_clang_report_name: true)     # run tests and generate reports
```
Comment (stpopa) [inline fastlane/Fastfile]: please discard this thing

Comment (mpetrenco) [reply fastlane/Fastfile]: Done.

File: Source/ApplePay/ApplePayConfiguration.m (hunk 1 above)
Comment (mpetrenco) [reply Source/ApplePay/ApplePayConfiguration.m]: Done.

**Pull request closed without merging**